src/app/pages/quickMaps/pages/biomarkers/biomarker.component.ts

```diff
@@ -25,7 +24,21 @@ export class BiomarkerComponent implements OnInit {
   public stopEvent: EventEmitter<GridsterItem> = new EventEmitter<GridsterItem>();
   public overlapEvent: EventEmitter<GridsterItem> = new EventEmitter<GridsterItem>();
 
-  constructor() { }
+  private readonly defaultWidgetHeight = 4;
+  private readonly defaultWidgetWidth = 6;
+
+  private dataLevelWidgetTypesMap: Map<DataLevel, Array<BiomarkerWidgets>> = new Map([
+    [DataLevel.COUNTRY, [
```

Comment:
@bgsandan will the biomarker page have 2 modes for data levels of "household" and "country", like the diet page does?

Reply:
I don't believe so.  Think its more of a one shot thing

Reply:
@DanielLyleWilliamWarren Looks like this can be simplified then, as it doesn't have to account for two alternative displays.